qxf2_scheduler/templates/edit-interviewer.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
```

Comment:
All common code between this file and add interviewer should go into templates and you should simply reference them here. You can look at either the training database application or wisdomofreddit as an example.